src/i_endoom.c

```diff
@@ -38,13 +38,10 @@ void I_Endoom(byte *endoom_data)
     int indent;
 
     // Set up text mode screen
+    TXT_PreInit(I_GetSDLWindow(), I_GetSDLRenderer());
 
     TXT_Init();
 
-    TXT_SetWindowTitle(PACKAGE_STRING);
```

Comment:
Where has this been moved to?

Reply:
Well, if ENDOOM appears in the same window as the game, it won't need to set a new window title.